### PR TITLE
Restyling best practices

### PR DESCRIPTION
**Pull request by geurts**
Restyling best practices, but also removed a last-of-type margin stye that was getting in the way, since I couldn't see how it was necessary.

### CODE REVIEW

File: pages/_includes/css/page-content.css [hunk 1]
```diff
@@ -19,19 +19,48 @@
...
+}
+div.d2l-daylight-dos, div.d2l-daylight-donts {
+	flex: 50%;
+	border: 1px solid var(--d2l-color-gray-10);
+	border-radius: .5rem;
+	background-color: white;
+	padding: .9rem 1.5rem 1.5rem 1.5rem;
+}
+.d2l-daylight-dos > :is(h1, h2, h3, h4, h5, h6).d2l-heading-3,
+.d2l-daylight-donts > :is(h1, h2, h3, h4, h5, h6).d2l-heading-3 {
```
Comment (margaree) [inline pages/_includes/css/page-content.css:39]: is the `:is` needed or can that just be `.d2l-daylight-dos > .d2l-heading-3` etc

File: .eleventy.cjs [hunk 2]
```diff
@@ -261,10 +261,13 @@ module.exports = function(eleventyConfig) {
 			const splitStart = content.split('<!-- docs: start ');
 			const splitEnd = splitStart[1].split(' -->');
 			const contentClass = `d2l-daylight-${splitEnd[0].replace(/ /g, '-')}`;
-			return `<div class="${contentClass}">`;
+			const heading = contentClass === 'd2l-daylight-dos' ? '<h3 class="d2l-heading-3">Do</h3>' : contentClass === 'd2l-daylight-donts' ? '<h3 class="d2l-heading-3">Don&apos;t</h3>' : '';
+			return `<div class="${contentClass}">` + heading;
 		} else return defaultHtmlRule(tokens, idx, options, env, slf);
 	};
 
+// BROKEN ... after adding the explicit "don't" case 👆
```
[the remark this answers is not in the document]
Comment (geurts) [reply .eleventy.cjs:269]: Oops, I think that was a note I left myself while troubleshooting something, thanks for catching it

File: pages/_includes/css/page-content.css [hunk 3]
```diff
@@ -19,19 +19,48 @@
...
+	.d2l-daylight-best-practices {
+		flex-direction: column;
+	}
+}
+div.d2l-daylight-dos, div.d2l-daylight-donts {
+	flex: 50%;
+	border: 1px solid var(--d2l-color-gray-10);
+	border-radius: .5rem;
+	background-color: white;
+	padding: .9rem 1.5rem 1.5rem 1.5rem;
```
[the remark this answers is not in the document]
Comment (geurts) [reply pages/_includes/css/page-content.css:36]: Gotcha, I'll do that!

File: pages/_includes/css/page-content.css (hunk 1 above)
Comment (geurts) [reply pages/_includes/css/page-content.css:39]: I can't remember why I had to do that but I think I found an issue the other way ... will try again and remove if it I can

File: pages/_includes/css/page-content.css [hunk 4]
```diff
@@ -19,19 +19,48 @@
...
+	margin: 0 0 .8rem 0;
+}
+.d2l-daylight-dos > .d2l-heading-3::before,
+.d2l-daylight-donts > .d2l-heading-3::before {
+	position: relative;
+	top: .1rem;
+	padding-right: .6rem;
+}
+.d2l-daylight-dos > .d2l-heading-3::before {
+	content: url("/img/check-circle-tier-1.svg");
```
[the remark this answers is not in the document]
Comment (geurts) [reply pages/_includes/css/page-content.css:49]: Okay thanks ... hopefully just an issue on the test site but not sure 😬

File: pages/_includes/css/page-content.css [hunk 5]
```diff
@@ -19,19 +19,48 @@
...
+.d2l-daylight-best-practices ul {
+	margin: 0;
+	padding: 0 1rem;
+}
+.d2l-daylight-best-practices li,
+.d2l-daylight-best-practices ul ul {
+	margin: 0.3rem 0;
+}
+
+/*
```
[the remark this answers is not in the document]
Comment (geurts) [reply pages/_includes/css/page-content.css:63]: I'll check ..

File: pages/_includes/css/page-content.css (hunk 1 above)
Comment (geurts) [reply pages/_includes/css/page-content.css]: Can't see any problems caused by removing the :is pseudoselector so I've removed it 👍

File: pages/_includes/css/page-content.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (margaree) [reply pages/_includes/css/page-content.css:49]: I tried on the local prod and it worked fine and looks fine in the local build, so we can just check after you merge and fix if needed, it isn't a huge deal if it's broken for a short time.

File: .eleventy.cjs [hunk 6]
```diff
@@ -261,7 +261,10 @@ module.exports = function(eleventyConfig) {
 			const splitStart = content.split('<!-- docs: start ');
 			const splitEnd = splitStart[1].split(' -->');
 			const contentClass = `d2l-daylight-${splitEnd[0].replace(/ /g, '-')}`;
-			return `<div class="${contentClass}">`;
+			const heading = '';
```
Comment (margaree) [inline .eleventy.cjs:264]: ugh my bad, sorry! Should be:
```suggestion
			let heading = '';
```

**Review by margaree — APPROVED** [review]
LGTM!

**Pull request merged**